packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -750,6 +750,14 @@ class AnimationController extends Animation<double>
     return result;
   }
 
+  /// Update the simulation without restarting the animation.
+  ///
+  /// The current simulation will be replaced with the provided [Simulation].
```

Comment:
Also, it may be useful to add a little color to this doc describing when this API could be useful (and when not - with links to other API methods that may be more appropriate).